examples/tf-s3-consumer/pom.xml

```diff
@@ -4,15 +4,15 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <properties>
-    <tf-maven-version>0.8.2</tf-maven-version>
+    <tf-maven-version>0.9.1</tf-maven-version>
   </properties>
   <artifactId>tf-s3-consumer</artifactId>
   <groupId>com.deliveredtechnologies.example.maven.tf</groupId>
   <version>1.0</version>
   <repositories>
     <repository>
-      <id>local</id>
```

Comment:
Why did you change it to sonatype's repo? Are you using SNAPSHOT artifacts from there?
IDK if it really matters.